s3/src/main/scala/org/allenai/pipeline/s3/CreateCoreArtifacts.scala

```diff
@@ -0,0 +1,36 @@
+package org.allenai.pipeline.s3
+
+import org.allenai.pipeline.{ Artifact, ArtifactFactory, CreateCoreArtifacts => CreateCoreFileArtifacts, Pipeline => basePipeline, UrlToArtifact }
+
+import scala.reflect.ClassTag
+
+import java.net.URI
+
+/** Created by rodneykinney on 5/22/15.
+  */
+object CreateCoreArtifacts {
```

Comment:
S3 support is in a separate sub-project.  Users that declare the S3 sub-project as a dependency can mix in the `S3Pipeline` trait into their pipeline classes, which gives them the ability to create Artifacts from s3: URLs.